gulp-tasks/build-packages.js

```diff
@@ -0,0 +1,115 @@
+const gulp = require('gulp');
+const path = require('path');
+const fs = require('fs-extra');
+const oneLine = require('common-tags').oneLine;
+const rollup = require('rollup-stream');
+const source = require('vinyl-source-stream');
+const sourcemaps = require('gulp-sourcemaps');
+const rename = require('gulp-rename');
+const buffer = require('vinyl-buffer');
+
+const constants = require('./utils/constants');
+const packageRunnner = require('./utils/package-runner');
+const logHelper = require('./utils/log-helper');
+const pkgPathToName = require('./utils/pkg-path-to-name');
+
+/**
+ * To test sourcemaps are valid and working, use:
+ * http://paulirish.github.io/source-map-visualization/#custom-choose
+ */
+
+const buildPackage = (packagePath, buildType) => {
+  const srcPath = path.posix.join(packagePath, 'src');
```

Comment:
Since we're not using `gulp.src()` here, and therefore not using globs, shouldn't this be `path.join()` instead of always using POSIX paths?